src/pool/cli.c

```diff
@@ -469,7 +469,7 @@ dc_pool_local_open(uuid_t pool_uuid, uuid_t pool_hdl_uuid,
 	}
 
 	/** allocate and fill in pool connection */
-	pool = pool_alloc();
+	pool = pool_alloc(pool_map_target_nr(map));
```

Comment:
seems should pass tgt_nr + domain_nr + node_nr

Reply:
ah, right.